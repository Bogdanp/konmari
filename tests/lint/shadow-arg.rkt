#lang racket/base

(define method "GET")

(define (request [method method])
  (void))
