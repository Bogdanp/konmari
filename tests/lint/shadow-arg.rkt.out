/shadow-arg.rkt:5:10:warning:identifier 'request' is never used
/shadow-arg.rkt:5:19:warning:identifier 'method' is never used
/shadow-arg.rkt:5:19:warning:identifier 'method' shadows an earlier binding
